src/pubsub/ua_pubsub_keystorage.c

```diff
@@ -592,26 +600,30 @@ callGetSecurityKeysMethod(UA_Client *client) {
     sksClientContext *ctx = (sksClientContext *)client->config.clientContext;
 
     UA_Variant *inputArguments = (UA_Variant *)UA_calloc(3, (sizeof(UA_Variant)));
```

Comment:
As this is an array of known size, allocate it statically.
So you also do not have to check for malloc failing.

Reply:
> 



> Please add this to the CI with memcheck enabled.

> Great! Comments inline. Please add this to the CI with memcheck enabled. So more problems are caught automatically.

unfortunately the sks_client test is also failing due to the similar issue as mentioned in #5611  & #5634 . That is why I am unable to add memcheck to the CI at the moment. May be when this is fixed. Can I add this in a new PR when #5611 is resolved?

`72: check_pubsub_sks_client: /home/muddasir/work/Projects/DronEdge/sks_open62541/open62541/arch/common/ua_timer.c:261: UA_Timer_process: Assertion `!UA_TimerTree_ZIP_MIN(&t->tree) || UA_TimerTree_ZIP_MIN(&t->tree)->nextTime > nowMonotonic' failed.
72: Aborted`